src/dodal/devices/attenuator/filter.py

```diff
@@ -0,0 +1,5 @@
+from ophyd import Component, Device, EpicsSignalRO
+
+
+class AtteunatorFilter(Device):
```

Comment:
Should: This is so small I would actually just put it in the attenuator file then move them out of the folder

Reply:
There are other at least two other PVs which use :FILTER{number} which I haven't needed to access yet. Better to leave the class in for if these PVs are added? Or remove it now and maybe readd later?